src/SDKs/CognitiveServices/CognitiveServices.Tests/Tests/CognitiveServicesAccountTests.cs

```diff
@@ -156,14 +155,14 @@ public void CognitiveServicesAccountListByResourceGroupTest()
                 // Create resource group
                 var rgname = CognitiveServicesManagementTestUtilities.CreateResourceGroup(resourcesClient);
 
-                var accounts = cognitiveServicesMgmtClient.CognitiveServicesAccounts.ListByResourceGroup(rgname);
+                var accounts = cognitiveServicesMgmtClient.Accounts.ListByResourceGroup(rgname);
```

Comment:
@shahabhijeet I have updated all nuget packages for test project though for ResourceManager (Microsoft.Azure.ResourceManager) I think we have used the latest (1.1.0-preview).  I don't see update option for it.